### DISCUSSION TITLE
Wineskin and Winery ask for permission to Desktop, Documents and Downloads. Why?

### DISCUSSION THREAD

**Discussion by ovvldc** [The-Wineskin-Project/WineskinServer#569] [General]
Why do Winery and Wineskin need access to these three folders, ~/Desktop, ~/Documents and ~/Downloads? I am using MacOS Sonoma on an M3 Macbook Air.

**Comment by emendelson** [comment]
They are mapped to folders in the emulated Windows system. Look at the Options tab, then Map User Mac OS X folders etc.

**Comment by ovvldc** [comment]
I remember now, thanks! It had been a while since i even looked at those..
